app/src/main/java/org/schabi/newpipe/views/NewPipeEditText.java

```diff
@@ -0,0 +1,60 @@
+package org.schabi.newpipe.views;
+
+import android.content.Context;
+import android.text.Selection;
+import android.text.Spannable;
+import android.util.AttributeSet;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+import androidx.appcompat.widget.AppCompatEditText;
+
+import org.schabi.newpipe.util.external_communication.ShareUtils;
+
+public class NewPipeEditText extends AppCompatEditText {
```

Comment:
``NewPipeTextView`` and ``NewPipeEditText``-methods are nearly identical.
Can we de-deuplicate the changes e.g. into an interface or a static helper class

Reply:
I don't think so and I think doing this will create a code which will be hard to understand.

However, I created a new class that I named `NewPipeTextViewHelper` which merges the sharing process of the two classes into one.